make/langtools/netbeans/nb-javac/pom-nb-javac.xml

```diff
@@ -0,0 +1,41 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+  http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>eu.doppel-helix.com.github.oracle.nb-javac</groupId>
```

Comment:
Could this line be replaced with a `TEMPLATE` and processed just like the `<version/>` tag in the build script? The publishing script would then have to be invoked with `ant -DgroupId=eu.doppel-helix.com.github.oracle.nb-javac` (or some other group ID). As a benefit, the whole publishing process would be exactly recorded in the repository and anyone with proper OOSRH account could replay it.

Reply:
Please see my comment on the developer section.

Reply:
> ... I don't intent to make the groupId a variable. ... the developers will be pretty constant 
> and the same is true for the group id.

I see. Then your PR needs to be modified before it gets to `origin/master`. Can I push my preferred changes into this PR or shall I create my own and use bce95d8 as base?

Reply:
Sure - this was intended to get the process started and demonstrate, that it is possible with too much fuzz. Feel free to improve, it's appreciated.

Reply:
There is [PR #1](https://github.com/matthiasblaesing/nb-javac/pull/1) with my changes to make `groupId` pluggable.